apps/web/src/pages/index.tsx

```diff
@@ -1,23 +1,71 @@
 import { Layout } from "../helper/layout";
 import type { ReactElement } from "react";
+import Image from "next/image";
+import Link from "next/link";
 
 export default function Landing() {
   return (
-    <>
+    <div className="flex flex-col h-[calc(100vh-6rem)] justify-between content-start gap-9">
       <div>
-        <h1 className="text-9xl font-sora">
-          <span>SK</span>
-          <span className="text-techdev-gradient-radial">TechDev</span>
-        </h1>
-        {/* <Button variant="contained" color="primary">
-          Join us
-        </Button> */}
+        <div>
+          <h1 className="text-9xl font-sora">
+            <span>SK</span>
+            <span className="text-techdev-gradient-radial">TechDev</span>
+          </h1>
+        </div>
+        <div>
+          <h2 className="text-5xl font-sora leading-[3.55rem] w-[60%]">
+            Technology <br /> meets entertainment.
+          </h2>
+          <div className="flex space-x-2 mt-6">
+            <Link href={"/contact"}>
+              <a>
+                <button className="bg-light-primary text-light-on-primary py-2 px-6 rounded-full dark:bg-dark-primary dark:text-dark-on-primary cursor-pointer hover:opacity-80 transition-all duration-300">
```

Comment:
As of now I have not make one since this pr is suppose to be a quick check on how quickly I can implement a page to set date line. I would be making components later on in the development

Reply:
I don't agree with this. A Landing page is the face of the website that requires the most love and care and is not representative of how long it would actually take the code the entire website. I also think creating components before making the pages would speed up development, so this "check" is even less representative.